src/Editmode.jsx

```diff
@@ -1,23 +1,28 @@
 // @ts-check
```

Comment:
We need to re-investigate splitting `Editmode.jsx` into 2 files:

- `Editmode.jsx` for web
- `Editmode.native.jsx` for `react-native`